src/components/Checkout/KivaCardRedemption.vue

```diff
@@ -265,21 +264,26 @@ div.columns .kiva-card-entry-wrapper {
 
 	@include breakpoint(medium) {
 		margin: rem-calc(15) 0;
+		height: rem-calc(38);
 	}
 
 	@include breakpoint(large) {
+		width: rem-calc(200);
 		font-size: $normal-text-font-size;
 		float: left;
-		height: rem-calc(37);
 		margin: rem-calc(15);
 	}
 }
 
 .help-lightbox {
 	margin: 1.2rem 0;
+	position: relative;
```

Comment:
Why do we need to override global styles here? Should these changes be made to the lightbox?

Reply:
@mcstover made the change in the new cleanup branch im working in, see this commit: https://github.com/kiva/ui/commit/8e64bf1efb3dc4abbfc5824dbe0bf5ad6f163377

Reply:
cool carry on